src/main/java/frc/robot/commands/ChargeStationBalance.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.subsystems.DriveTrain;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ChargeStationBalance extends CommandBase {
+
+  private final Rotation2d BALANCED_ERROR = Rotation2d.fromDegrees(2.5); //error for what counts as balanced
+  private final double BALANCED_DEGREES = 0;
+  private final double BALANCE_KP = 0.02; //change to control how fast robot drives during balancing
+  private final double MAX_MPS = 0.75;
+  private final double BALANCE_SECONDS = 1; //how many seconds the robot has to be balanced before stopping
```

Comment:
Oh, I forgot to mention: constants should also marked "static". Otherwise there is a copy in each of the instances.